oxidation/crates/parsec_client_types/tests/test_local_manifest.rs

```diff
@@ -992,48 +992,48 @@ fn chunk_is_block() {
         access: None,
     };
 
-    assert_eq!(chunk.is_pseudo_block(), true);
-    assert_eq!(chunk.is_block(), false);
+    assert!(chunk.is_pseudo_block());
```

Comment:
Not sure about what we gain doing this, the assert_eq form seems more readable to me compared to just the ! character

@vxgmichel @TimeEngineer any though on this ?

Reply:
I agree, but acceptable

Reply:
It's the default configuration of clippy concerning the `assert` vs `assert_eq` on boolean